doc/src/build/sui-local-network.md

```diff
@@ -2,58 +2,159 @@
+            "transferTxDigest": "7sp4fFPH2WaUgvN43kjDzCpEhKfifqjx5RTki74y8T3E"
+        },
+        {
+            "amount": 200000000,
+            "id": "0x31d003ade00675d1ab82b225bfcceaa60bb993f5d90e9d0aa88f81dc24ec14d6",
+            "transferTxDigest": "7sp4fFPH2WaUgvN43kjDzCpEhKfifqjx5RTki74y8T3E"
+        },
+        {
+            "amount": 200000000,
+            "id": "0x98cbdc93ae672110f91bc0c39c0c87bc66f36984c79218bb2c0bac967260970c",
+            "transferTxDigest": "7sp4fFPH2WaUgvN43kjDzCpEhKfifqjx5RTki74y8T3E"
+        },
+        {
+            "amount": 200000000,
+            "id": "0xba66aee6289cc6d0203c451bea442ad30d4cfe699e50b36fed0ff3e99ba51529",
+            "transferTxDigest": "7sp4fFPH2WaUgvN43kjDzCpEhKfifqjx5RTki74y8T3E"
+        },
+        {
+            "amount": 200000000,
+            "id": "0xd9f0b521443d66227eddc2aac2e16f667ca9caeef9f1b7afb4a6c2fc7dcb58d8",
+            "transferTxDigest": "7sp4fFPH2WaUgvN43kjDzCpEhKfifqjx5RTki74y8T3E"
+        }
+    ],
+    "error": null
+}
+```
 
-Run the following command to start the local Sui network, assuming you
-accepted the default location for configuration:
+### Accessing Full node
 
-```shell
-sui start
+You can access your Full node using cURL:
+
+```bash
+curl --location --request POST 'http://127.0.0.1:9000' \
+--header 'Content-Type: application/json' \
+--data-raw '{
+  "jsonrpc": "2.0",
+  "id": 1,
+  "method": "sui_getTotalTransactionNumber",
+  "params": []
+}'
 ```
 
-This command looks for the Sui network configuration file
-`network.yaml` in the `~/.sui/sui_config` directory. If you used a different directory when you ran `genesis`, use the `--network.config` argument to specify the path to that directory when you start the network.
+If successful, the return resembles the following:
+
+```bash
+{
+    "jsonrpc": "2.0",
+    "result": 168,
+    "id": 1
+}
+```
+
+## Setup local Sui Explorer
+
+While [https://explorer.sui.io/?network=local](https://explorer.sui.io/?network=local) is compatible with the local network, it might not have all the latest features that are available in the `main` branch of the Sui repository. To run `explorer` locally, open a Terminal or Console window in the `sui` directory (install [pnpm](https://pnpm.io/installation) first if you don't already have it):
+
+```bash
+pnpm explorer dev
+```
+
+After running the command, you can open a browser to [http://localhost:3000/](http://localhost:3000/) to access your local version of Sui Explorer.
+
+For more details, see [https://github.com/MystenLabs/sui/tree/main/apps/explorer](https://github.com/MystenLabs/sui/tree/main/apps/explorer).
+
+## Set up local Sui Wallet
+
+Similar to local Sui Explorer, you can also setup a local Sui Wallet. Open a Terminal or Console window or tab at the `sui` root directory and use the `wallet start` command (install [pnpm](https://pnpm.io/installation) first if you don't already have it):
+
+```bash
+pnpm wallet start
+```
+
```

Comment:
can we add something like

```
Tips: You can set the default environment to be local network via https://github.com/MystenLabs/sui/tree/main/apps/wallet#environment-variables so that you don't need to switch network manually
```